src/app/leverage/components/stats/leverage-selector-container.tsx

```diff
@@ -146,10 +148,12 @@ export function LeverageSelectorContainer() {
       <TokenSelector selectedToken={indexToken} />
       <StatsMetric
         className='hidden w-20 sm:flex'
-        isLoading={isFetchingStats}
+        isLoading={isFetchingQuickStats}
         label='24h Change'
-        value={`${formatAmount(navchange, 2)}%`}
-        overrideLabelColor={navchange >= 0 ? 'text-ic-green' : 'text-ic-red'}
+        value={`${formatPercentage(data.token.navchange)}%`}
```

Comment:
```suggestion
        value={formatPercentage(data.token.navchange)}}
```
already adds the percentage sign to the end

Reply:
🦅 👀